Makefile

```diff
@@ -37,6 +39,14 @@ help:
 # http://marmelab.com/blog/2016/02/29/auto-documented-makefile.html
 	@grep -E '^[a-zA-Z0-9_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
+build:
+env: ## Make a dev environment
+	-conda env create --file requirements.yml --name $(ENV)
+	-conda env config vars set PYTHONPATH=$(PWD) --name $(ENV)
+
+activate: ## Print instructions to activate the virtualenv (default: enterprise-gateway-dev)
+	@echo "Run \`$(SA) $(ENV)\` to activate the environment."
```

Comment:
Not sure how useful this will be, but there may be some that find it helpful.

Reply:
I found it helpful myself as I was getting started with this project since I don't have a lot of familiarity with different python environment tools

Reply:
Sounds good - perfect feedback.